src/components/Layout/Header/ActionCenter/ActionCenter.tsx

```diff
@@ -33,26 +40,69 @@ export const ActionCenter = memo(() => {
   const { isOpen, onOpen, onClose } = useDisclosure()
 
   useSwapActionSubscriber({ onDrawerOpen: onOpen })
+  useLimitOrderActionSubscriber({ onDrawerOpen: onOpen })
 
   const translate = useTranslate()
+  const [orderToCancel, setOrderToCancel] = useState<OrderToCancel | undefined>(undefined)
 
-  const actions = useAppSelector(selectWalletActionsSorted)
+  const actions = useAppSelector(selectInitializedActionsByUpdatedAtDesc)
```

Comment:
same re: naming as in the previous PR

Reply:
wrong rebase [ae45489](https://github.com/shapeshift/web/pull/9626/commits/ae454893cdc65e03db925d010f1ec8b8482fc39d)